chirp/drivers/mml_jc8810.py

```diff
@@ -1239,3 +1267,38 @@ class HI8811Radio(RT470LRadio):
     # The HI-8811 support in this driver is curretnly based upon v1.17
     # firmware.
     # ==========
+
+
+@directory.register
+class UVA37Radio(JC8810base):
+    """Anysecu UV-A37"""
+    VENDOR = "Anysecu"
+    MODEL = "UV-A37"
+
+    # ==========
+    # Notice to developers:
+    # The UV-A37 support in this driver is curretnly based upon v1.24
```

Comment:
"currently" is misspelled.

Reply:
You've gotta love copy-and-paste for making it so easy to duplicate mistakes. Thanks for fixing.